src/test/java/DecodeTest.java

```diff
@@ -0,0 +1,73 @@
+import decoder.Decoder;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Validate that decoding the protobuf file to json works properly. */
+@RunWith(JUnit4.class)
+public final class DecodeTest {
+  private static String inputFile;
+  private static String outputFile;
+  private static InputStream input;
+  private static OutputStream output;
+
+  @Before
+  public void setUp() {
+    inputFile = "sampleProto.sstable";
```

Comment:
just to clarify, is all protobuf files in .sstable format?

Reply:
I believe that's the extension, it doesn't really matter as its just raw binary data